modules/security_group/main.tf

```diff
@@ -7,3 +7,23 @@ resource "aws_security_group" "rds" {
     Name = "${var.namespace}-rds-sg"
   }
 }
+
+resource "aws_security_group" "alb" {
+  name        = "${var.namespace}-alb-sg"
+  description = "ALB security group"
+  vpc_id      = var.vpc_id
+
+  tags = {
+    Name = "${var.namespace}-alb-sg"
+  }
+}
+
+resource "aws_security_group_rule" "alb_ingress_http" {
+  type              = "ingress"
+  security_group_id = aws_security_group.alb.id
+  protocol          = "tcp"
+  from_port         = 80
+  to_port           = 80
+  cidr_blocks       = ["0.0.0.0/0"]
+  description       = "From HTTP to ALB"
+}
```

Comment:
Do we need to add another security_group_rule for HTTPS here, ie `to_port = 443`.

I see there's a similar comment `// TODO: Add HTTPS listener if still have time` related to a listener though? 🤔

Reply:
I think we don't need to duplicate the TODO, and it's also not in the requirement. If I still have time, I will add HTTPS  configuration and security rule should be added as a mandatory part 👍